### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by aeon9300**
**My portfolio project.**

A Skeleton Version

- HTML
- CSS
- LINTERS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <nav>
+                <div class="menu1">
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                </div>
+            </nav>
+            </div>
+            
+        </header>
```
Comment (MrBrN197) [inline index.html:28]: - [x] Your design is really good 👍. Please watch out for the spacings in order to match the Figma template perfectly. If you look at your design the bottom spacing on the header is too much. please address this issue.
## Your Design
image
## Figma Design 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <li class="social-2"><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
+        <li class="social-3"><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
+        <li class="social-4"><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
+        <li class="social-5"><a href="#"><i class="fab fa-medium-m"></i></a></li>
+        </ul>
+
+    </div>
+    
+    <div class="hero-image">
+        <img src="images/background-mobile.svg" alt="" />
```
Comment (MrBrN197) [inline index.html]: - [x] Hi you've done well with the design of the headline section 👍. I'd like you, however, to use CSS for the background image using the `background-image` CSS property. this is part of the requirements. so please implement this background image using CSS 🙏

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @aeon9300  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Design really well matches the Figma template.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (aeon9300) [reply index.html:28]: Spacing now matches Figma Template

File: index.html (hunk 2 above)
Comment (aeon9300) [reply index.html]: Background image now done in CSS

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @aeon9300 ,  

Your project is complete! Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MrBrN197) [reply index.html]: Thanks 👍

File: index.html (hunk 1 above)
Comment (MrBrN197) [reply index.html:28]: Well done 👍

**Pull request merged**